actions/admin_actions.jsx

```diff
@@ -347,3 +347,13 @@ export function unregisterAdminConsolePlugin(pluginId) {
         });
     };
 }
+
+export async function testSiteURL(success, error, state) {
```

Comment:
if you convert this to a thunk action you don't need to pass in this `state` variable.

Reply:
The `state` here refers to the local state of component and not the global state.  I just renamed this to `config` since it's a config from local component state.

Reply:
@saturninoabril do you mind just passing in the siteURL instead, i think that would be the cleanest. Otherwise this action becomes hard to re-use anywhere else.